hax/hax/util.py

```diff
@@ -1403,15 +1406,17 @@ def node_to_drive_fid(self, node_name: str, drive: str):
         # 4. find drive name in the json value and extract sdev fid from the
         #    key 0x6400000000000001:0x2c
         # 5. Create sdev fid from sdev fid key.
-        process_items = self.kv.kv_get(f'm0conf/nodes/{node_name}/processes',
-                                       recurse=True)
-        for x in process_items:
-            if '/ios' in x['Key']:
-                fidk_ios = x['Value']
-        ios_fid = create_service_fid(int(fidk_ios))
+        # process_items = self.kv.kv_get(f'm0conf/nodes/{node_name}/processes',
+        #                                recurse=True)
+        # for x in process_items:
+        #     if '/ios' in x['Key']:
+        #         fidk_ios = x['Value']
+        # ios_fid = create_service_fid(int(fidk_ios))
+        LOG.debug('node_to_drive_fid: node_name: %s drive', node_name, drive)
         sdev_items = self.kv.kv_get('m0conf/nodes', recurse=True)
         for x in sdev_items:
-            if f'/{ios_fid}/' in x['Key']:
+            # if f'/{ios_fid}/' in x['Key']:
+            if '/sdevs/' in x['Key']:
```

Comment:
Is the above description of the algorithm still correct?

Reply:
Updated the comment and deleted the commented statements.
Yes, this is expensive due to our kv schema. I have another idea to improve such lookups. Will be doing it in a separate task.
Thanks.